### ISSUE TITLE
Feature: Ultravisor support

### ISSUE THREAD

**Issue by mpe** [linuxppc/issues#270]
v6: https://patchwork.ozlabs.org/project/linuxppc-dev/list/?series=126630

Waiting on ELF note patch to be posted.

**Comment by mpe** [comment]
To be applied to topic/ppc-kvm.

**Comment by mpe** [comment]
Merged into next as of linuxppc/linux@9044adca782c.

**Comment by mpe** [comment]
Secure guest support merged into next as of linuxppc/linux@bf75a8db72e9.

**Comment by mpe** [comment]
Still pending series from Bharata:

https://patchwork.ozlabs.org/project/linuxppc-dev/list/?series=132453

Will be merged via the kvm-ppc tree.

**Comment by mpe** [comment]
Merged as of torvalds/linux@013a53f2d25a :100:
